code/modules/admin/verbs/randomverbs.dm

```diff
@@ -172,7 +172,7 @@
 
 	log_admin("[key_name(usr)] has [muteunmute] [key_name(whom)] from [mute_string]")
 	message_admins("[key_name_admin(usr)] has [muteunmute] [key_name_admin(whom)] from [mute_string].")
-	if(C)	C << "You have been [muteunmute] from [mute_string]."
+	if(C)	C << "You have been [muteunmute] from [mute_string] by [usr.client.ckey]."

```

Comment:
everything else uses it in this file and I don't feel like refactoring the FUCKING MESS that is admin code